### ISSUE TITLE
 bot.sendMessage doesn't work

### ISSUE THREAD

**Issue by DConstantine9** [yagop/node-telegram-bot-api#768]
Hey guys I have following issue:
when I'm trying to send message to user i have an error:

fn = function () { throw arg; };
StatusCodeError: 400 - {"ok":false,"error_code":400,"description":"Bad Request: message text is empty"}

My code:
`bot.on('message', function(msg) {
  console.log(msg);
  bot.sendMessage(msg.chat.id, "hello")
});`

**Comment by kamikazechaser** [comment]
Are you sending "hello" or some other text? Sometimes non-ASCII text may give this error. Post the exact message that you are trying to send.

**Comment by 97thjingba** [comment]
@DConstantine9 
Post your complete code

**Comment by DConstantine9** [comment]
@97thjingba 
Here it is:
`const telegram = require('telegram-bot-api');

const bot = new telegram({
  token: 'here is my token',
  updates: {
    enabled: true
  }
});

bot.on('message', function(message) {
  console.log(message);
  bot.sendMessage(message.chat.id, "hello")
});`

**Comment by DConstantine9** [comment]
@kamikazechaser 
it doesn't want to send any message

**Comment by 97thjingba** [comment]
@DConstantine9 
const bot = new TelegramBot(token, {polling:true});
try this

**Comment by DConstantine9** [comment]
@97thjingba 
It doesn't work. The same error

**Comment by saeedhei** [comment]
You Should use:``` const telegram = require(node-'telegram-bot-api')```

**Comment by kamikazechaser** [comment]
@DConstantine9 Looks like you are using the wrong library.

**Comment by DConstantine9** [comment]
Well, I realized that node-telegram-bot-api and telegram-bot-api are 2 different apis.  was confused by this. I tried diffenerent documentation and it worked. 

But now it drops this error:

node-telegram-bot-api deprecated Automatic enabling of cancellation of promises is deprecated.
In the future, you will have to enable it yourself.
See https://github.com/yagop/node-telegram-bot-api/issues/319. module.js:652:30

And this page that they suggest to visit doesn't exist.

Thank you guys for help!
